website/layouts/partials/head-custom.html

```diff
@@ -0,0 +1,53 @@
+<style type="text/css">
```

Comment:
I see there is a [`assets/css/custom.css`](https://github.com/basil/antora-default-ui-hugo-theme/blob/master/assets/css/custom.css) that can be overridden instead

Reply:
Thanks! Much better.